bundle/edu.gemini.itc.web/src/main/scala/edu/gemini/itc/web/ITCRequest.scala

```diff
@@ -275,19 +275,37 @@ object ITCRequest {
   def sourceDefinitionParameters(r: ITCRequest): SourceDefinition = {
     // Get the source geometry and type
     import SourceDefinition.Profile._
+
+    def magnitudeSystemFor(s: String) = s match {
+      case "MAG"                  => MagnitudeSystem.Vega
+      case "ABMAG"                => MagnitudeSystem.AB
+      case "JY"                   => MagnitudeSystem.Jy
+      case "WATTS"                => MagnitudeSystem.Watts
+      case "ERGS_WAVELENGTH"      => MagnitudeSystem.ErgsWavelength
+      case "ERGS_FREQUENCY"       => MagnitudeSystem.ErgsFrequency
```

Comment:
If these are the canonical string representations can we put them in `fromString(s:String): Option[Whatevs]` and `unsafeFromString(s: String): Whatevs` methods on the Companions?